src/shared/clients/codecatalystClient.ts

```diff
@@ -700,9 +700,16 @@ class CodeCatalystClientInternal {
                         await this.startDevEnvironment(args)
                     } catch (e) {
                         const err = e as AWS.AWSError
+                        // - ServiceQuotaExceededException: account billing limit reached
                         // - ValidationException: "… creation has failed, cannot start"
                         // - ConflictException: "Cannot start … because update process is still going on"
                         //   (can happen after "Update Dev Environment")
+                        if (err.code === 'ServiceQuotaExceededException') {
+                            throw new ToolkitError('Dev Environment failed: quota exceeded', {
+                                code: 'ServiceQuotaExceeded',
```

Comment:
You can omit `code`, it'll report the code of `cause` instead.

Reply:
wondering if we should be explicit here, so that the metrics have the same code, even cross-toolkit? otoh, if other toolkits don't do this then we could end up with _less_ comparable/consistent metrics.